src/lib_ccx/ccx_decoders_common.c

```diff
@@ -16,9 +16,7 @@ made to reuse, not duplicate, as many functions as possible */
 #include "ccx_dtvcc.h"
 
 #ifdef ENABLE_RUST
-// Include rust decoder
-extern void dtvcc_process_cc_data(struct dtvcc_ctx *dtvcc,
-				  const unsigned char *data);
+extern int dtvcc_process_cc_data(struct lib_cc_decode *dec_ctx, unsigned char *cc_data, int cc_count);
```

Comment:
I think it's safe to have this declaration all the time, ENABLE_RUST or not :-)

Reply:
That's a rust function, so it needs to be inside ENABLE_RUST